<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Btn_load">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lab_path">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>......................................................</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>功能</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>28</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>学号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>28</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>68</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>专业</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>68</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ledit_num">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ledit_major">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>65</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ledit_name">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>25</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ledit_gender">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>65</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_insert">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>插入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_query">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>奖励</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ledit_reward">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>107</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
